### ISSUE TITLE
"FIXED" test cases?

### ISSUE THREAD

**Issue by b4n** [universal-ctags/ctags#1593]
I wonder if those really should be marked as not broken anymore, or if the *expected.tags* file is just missing?

```
Summary (see CMDLINE.tmp to reproduce without test harness)
------------------------------------------------------------
  #passed:                                841
  #FIXED:                                 2
	parser-puppetManifest.r/puppet-tag
	parser-puppetManifest.r/puppet-tagged
```

**Comment by masatake** [comment]
I'm sorry that I have ignored the result.
I wondered how `tag` of puppet should be recorded and I just wanted to record the input for future work.
The result makes us confused. So I will update the misc/units to be able to show my intent.

**Comment by b4n** [comment]
> I wondered how |tag| of puppet should be recorded and I just wanted to
> record the input for future work.
> The result makes us confused. So I will update the misc/units to be able
> to show my intent.

Maybe just add a README in the meantime, there might not be any need for
a new category if it's not something common.

**Comment by masatake** [comment]
I see. I will add and commit directly to master branch.

**Comment by masatake** [comment]
Fixed in #1595 .
